src/styles/App.scss

```diff
@@ -1,7 +1,6 @@
-#root {
+/*#root {
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
   text-align: center;
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-}
\ No newline at end of file
+}*/
```

Comment:
So this code is just going to be commented out for now until we get our SCSS fully functional.